CHANGELOG.md

```diff
@@ -1,18 +1,30 @@
 # Changelog
 
-## unreleased for 3.0.0
-
-_Please add changes here that are only in the release-3.0 branch. These will be the source for the changelog when cutting the first release candidate for v3.0.0._
-
+## 3.0.0-beta.0 / 2024-09-05
 * [CHANGE] PromQL: Range selectors and the lookback delta are now left-open, i.e. a sample coinciding with the lower time limit is excluded rather than included. #13904
 * [CHANGE] Kubernetes SD: Remove support for `discovery.k8s.io/v1beta1` API version of EndpointSlice. This version is no longer served as of Kubernetes v1.25. #14365
 * [CHANGE] Kubernetes SD: Remove support for `networking.k8s.io/v1beta1` API version of Ingress. This version is no longer served as of Kubernetes v1.22. #14365
-
-## unreleased
-
+* [CHANGE] UTF-8: Enable UTF-8 support by default. Prometheus now allows all UTF-8 characters in metric and label names. The corresponding `utf8-name` feature flag has been removed. #14770
```

Comment:
It seems like #14770 is a different PR?

Reply:
Whoops, you're right. Fixed that and another PR ref in https://github.com/prometheus/prometheus/pull/14853/commits/1682d6192c128199bf1c96a333be466df0926dd7